code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -1007,14 +1007,14 @@
 
 /obj/effect/collapse/New()
 	..()
-	visible_message("<B><span class='danger'>The tendril writhes in pain and anger and the earth around it begins to split! Get back!</span></B>")
-	visible_message("<span class='danger'>A chest falls clear of the tendril!</span>")
+	visible_message("<span class='boldannounce'>The tendril writhes in fury as the earth around it begins to crack and break apart!</span>")
```

Comment:
I thought the explicit "get back!" was kind of important considering it's instant death